x-pack/plugins/ml/public/components/job_selector/_job_selector.scss

```diff
@@ -9,16 +8,16 @@
 
 .mlJobSelector__ganttBar {
   background-color: #79adda;
-  height: 14px;
+  height: $euiSizeM;
   border-radius: 2px;
 }
 
 .mlJobSelector__ganttBarBackEdge {
-  height: 18px;
+  height: $euiSize;
   border-left: 1px solid #d6d6d6;
   border-right: 1px solid #d6d6d6;
-  margin-bottom: -16px;
-  padding-top: 9px;
+  margin-bottom: -14px;
```

Comment:
Is there any way this could be a EUI sizing variable instead? Is 14px a must?

Reply:
Unfortunately, that's the only value I could make work for the ganttBar style.